### DISCUSSION TITLE
set external temperature from HA to luxtronic?

### DISCUSSION THREAD

**Discussion by alexander-jung** [BenPru/luxtronik#276] [Q&A] [answered]
enhancement request

Our temperature sensor for the external temperature is at an unfourtunate position: during may-occtober it gets direct sunlight and thus way too high temperatures.

I want to use other outside temperature sensors (e.g. from the ventilation intake) in the luxtronic instead. Is there a way i could *write* the external temperature to the luxtronic?

**Comment by AJediIAm** [comment, marked as answer]
To my knowledge, this is not possible. An accurate external temperature sensor is critical for proper and efficiency use of the heat pump. 

That said, you can set a lot of values which are related to the external temperature. You could adjust your heat curve values based on the observed delta between the external sensor and actual outside temperature. This is a dangerous rabbit hole to go down in though so recommend not to do it.

You can increase luxtronik_heating_room_temperature_impact_factor to compensate a little bit but it only goes so far. Still not a proper solution, but it should help a bit.

Best thing is to move the sensor to a good spot.
